templates/openwisp2/settings.py

```diff
@@ -165,12 +170,22 @@
         'schedule': timedelta(days=1),
         'args': ({{ openwisp2_notifications_delete_old_notifications }},),
     },
+    {% if openwisp2_monitoring %}
+    'run_checks': {
+        'task': 'openwisp_monitoring.check.tasks.run_check',
```

Comment:
it's run_checks, the docs of openwisp-monitoring was incorrect, see https://github.com/openwisp/openwisp-monitoring/issues/281